src/evaluation/evaluator.h

```diff
@@ -614,43 +572,428 @@ class Evaluator {
         m_ply--;
     }
 
-    void startTimeHandler()
-    {
-        bool wasStopped = m_isStopped.exchange(false);
-        if (!wasStopped) {
-            return; /* nothing to do here */
-        }
+    static uint8_t s_numSearchers;
+    static uint64_t s_nodes;
```

Comment:
Multiple threads can access this - not safe. 
I think it would be better to not make it static and instead accumulate each node count (avoid raciness and atomics)

Reply:
Sorted :) Now the evaluator owns the its own m_nodes and the evaluator does summation on `printScoreInfo`, which it now owns.